samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/smp_svr.c

```diff
@@ -0,0 +1,73 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+#ifdef ENABLE_SMP_SVR
```

Comment:
There's no need for this AFAICT. The file is only compiled if it is needed, i.e. if `CONFIG_BOOTLOADER_MCUBOOT=y`.